contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -1116,13 +1116,18 @@ export const ContentNode = new Resource({
         return Promise.reject('new lft value evaluated to null');
       }
 
-      let data = { parent, lft };
+      let data = { parent, lft, changed: true };
       let oldObj = null;
       return this.table
         .get(id)
         .then(node => {
           oldObj = node;
-          return this.table.update(id, data);
+          return this.table.update(id, data).then(() => {
+            // Set old parent to changed
+            if (oldObj.parent !== parent) {
+              return this.table.update(oldObj.parent, { changed: true });
+            }
+          });
         })
```

Comment:
Since `oldObj` exists in outer scope, you could break this new `.then()` block out to avoid nesting:
```js
.then(node => {
  oldObj = node;
  return this.table.update(id, data);
})
.then(() => {
  // Set old parent to changed
  if (oldObj.parent !== parent) {
    return this.table.update(oldObj.parent, { changed: true });
  }
})
```